# STAT 207 Lab 6 - Group Part - [5 total points] - Solutions

## Hypothesis Testing

Due: Tuesday, October 5, 11:59pm CST

## <u>Case Study</u>: Former STAT107 Student Analysis

In this group assignment, you will be using the same random sample of 6 students from lab_05_group to conduct a hypothesis test on your population parameter of interest. Remember, your population of interest in this case study is former STAT107 students from the last four semesters.

In this analysis you should come up with a value that you think your population parameter might be. In your hypothesis test, you will see if we have enough evidence to suggest that your population parameter is not equal to this value.

In [1]:
#Imports
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns; sns.set()
import random
from scipy.stats import norm
from statistics import mean

### 1. Hypotheses

First, set up your null and alternative hypotheses to evaluate these claims. You should use the appropriate notation to write these hypotheses. For any population parameter terms that you use, you should define what this parameter refers to.

$H_0: \mu=2$.

$H_A: \mu\neq 2$.

Where $\mu=$ number of 100 courses students have taken.

### 2. Conditions for Hypothesis Testing

Check the appropriate conditions for conducting this hypothesis test.

1. <u>Condition: The observations are independent</u>.
    
2. <u>Condition: Either n>30 OR the population distribution is normal.</u>.
    
    - The sample size doesn't meet the condition but the sample size is assinged to be 6 according to the instructions.

### 3. p-value

Calculate the p-value for this hypothesis test.

In [2]:
assumed = 2
mean = 3.333333
std = 1.632993

In [3]:
std_error=std/np.sqrt(6)
test_statistic=(mean-assumed)/std_error
test_statistic

1.999999698231605

In [5]:
pvalue=2*(1-norm.cdf(np.abs(test_statistic), loc=0, scale=1))
print(pvalue)

0.0455002964819029


### 4. Conclusion

Finally, use your p-value and a significance level of $\alpha=0.05$ to make a conclusion about your hypotheses.

Since 0.0455002964819029 < 0.05, we can reject the null hypothesis.

### <u>Contribution Report</u>: